seqr/utils/file_utils_tests.py

```diff
@@ -39,25 +39,23 @@ def test_get_gs_file_list(self, mock_logger, mock_subproc):
         self.assertEqual(str(ee.exception),  'A Google Storage path is expected.')
 
         process = mock_subproc.Popen.return_value
-        process.wait.return_value = -1
-        process.stdout = [b'-bash: gsutil: command not found.\n', b'Please check the path.\n']
+        process.communicate.return_value = None, b'-bash: gsutil: command not found.\nPlease check the path.\n'
```

Comment:
It is better to change `None` to `b''` because that is the value in the real world.